src/garage/experiment/local_tf_runner.py

```diff
@@ -340,73 +377,36 @@ def resume(self,
         assert self.train_args is not None, (
             'You must call restore() before resume().')
 
-        return self._train(
-            n_epochs=n_epochs or self.train_args.n_epochs,
-            n_epoch_cycles=n_epoch_cycles or self.train_args.n_epoch_cycles,
-            batch_size=batch_size or self.train_args.batch_size,
-            plot=plot or self.train_args.plot,
-            store_paths=store_paths or self.train_args.store_paths,
-            pause_for_plot=pause_for_plot or self.train_args.pause_for_plot,
-            start_epoch=self.train_args.start_epoch)
-
-    def _train(self,
-               n_epochs,
-               n_epoch_cycles,
-               batch_size,
-               plot,
-               store_paths,
-               pause_for_plot,
-               start_epoch=0):
+        if n_epochs:
+            self.train_args.n_epochs = n_epochs
```

Comment:
The decrease in code coverage is entirely due to these lines never being tested.
They're probably correct, but it's still a good practice to test them (in my opinion).
Out of principle, please add a test to `tests/garage/experiment/test_local_tf_runner.py` which sets all of these parameters.